src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
I'm open to other names, but I still feel it's important to be able to construct constraints using "JuMP syntax" without necessarily attaching them to models.